src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -15,9 +19,13 @@ public User login(String login, String password) throws AuthenticationException
     }
 
     private User findByLogin(String login) {
+        log.debug("Service level. Find by login was called with params: {}", login);
```

Comment:
Let's make our log messages informative. It's good to log the method entry, but since this is a debug level log, consider adding more context if possible.